### PR TITLE
feat(test): sets up testing for ossm plugin

### PR DESCRIPTION
**Pull request by bartoszmajsak**
Run the tests using the new target `test-ossm` or from the IDE.

Note: due to dependency hell we have to stick with Ginkgo v1 for the time being :(

### CODE REVIEW

File: pkg/kfapp/ossm/ossm_plugin_test.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+			if err := cli.List(context.Background(), namespaces); err != nil && !errors.IsNotFound(err) {
+				Fail(err.Error())
+			}
+			for _, namespace := range namespaces.Items {
+				if _, ok := namespace.ObjectMeta.Annotations["opendatahub.io/service-mesh"]; ok {
+					ns = append(ns, namespace)
+				}
+			}
+			return ns
+		}, timeout, interval).Should(HaveLen(1))
```
Comment (bartoszmajsak) [inline pkg/kfapp/ossm/ossm_plugin_test.go:57]: It seems you can't really use selectors for annotations :(

Comment (cam-garrison) [reply pkg/kfapp/ossm/ossm_plugin_test.go:57]: +1, sad : (

**Comment by cam-garrison** [comment]
Hey @bartoszmajsak, I don't see a reference to the new target `test-ossm` anywhere in the changes made - am I missing something or have some of the changes not been pushed?

**Review by cam-garrison — APPROVED** [review]
LGTM, successfully ran locally with the make target, issues with my IDE running tests but that is a config issue on my part.

**Pull request merged**